challenges.js

```diff
@@ -13,6 +12,13 @@ class Challenge {
     challengeCard.id = this.data.id;
     challengeCard.classList.add('challenges-container__challenge');
     challengeCard.classList.add(this.data.type);
+    if (this.data.type === 'online') {
+      challengeCard.innerHTML = '<i class="fa-solid fa-globe"></i>';
+    } else {
+      challengeCard.innerHTML = '<i class="fa-solid fa-building-user"></i>';
+    }
+
+    console.log(challengeCard.innerHTML);
```

Comment:
Don't forget to remove the below, if it only was there for testing. 😊

Reply:
Yes thank you, I will re-commit and remove it!

Reply:
Done!